### ISSUE TITLE
БЭМ (bem-tools) и паттерн MVC (и какой ни будь фреймворк)

### ISSUE THREAD

**Issue by Bumerang47** [bem/bem-forum-content-ru#331]
Здравствуйте. 
Очень нравится оптимизированные файлы после cборки в bem-tools. 
Сейчас хочу переверстать сайт, построенный на фреймворке Yii при помощи БЭМ. Но появляется вопрос, как использовать склеенную borschik'ом разметку (c трудом читаемую на глаз) в разных layout-ах и элементах view. Думаю это актуально при любом паттерне mvc и разных фреймворках. 
Дайте пожалуйста рекомендации, как лучше и правильней. Может есть специальные ключи для разбивки исходного файла при сборке.

**Comment by tadatuta** [comment]
@Bumerang47 я правильно понял, что речь именно про HTML? если так, то подойдет любой бьютифаер, например, https://www.npmjs.com/package/js-beautify#css-and-html

Есть готовый пакет для `ENB`: https://github.com/enb-make/enb-beautify
И древний пример технологии для `bem-tools`: https://github.com/tadatuta/gorod-mechty-www/blob/master/.bem/techs/html.js (в ней используется `htmltidy`, который тоже стоит заменить на какой-нибудь нативный модуль типа того же `js-beautify`).

**Comment by Bumerang47** [comment]
Да, речь скорее про HTML, т.к. с остальным проблем не должно возникнуть. Спасибо, буду пробовать.
